Packages/io.chainsafe.web3-unity/Runtime/Scripts/EVM/Token/Evm.cs

```diff
@@ -0,0 +1,189 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Numerics;
+using System.Text;
+using System.Threading.Tasks;
+using ChainSafe.Gaming.Evm.Providers;
+using ChainSafe.Gaming.Evm.Transactions;
+using ChainSafe.Gaming.UnityPackage;
+using ChainSafe.Gaming.Web3;
+using Nethereum.Hex.HexConvertors.Extensions;
+using Nethereum.Hex.HexTypes;
+using Nethereum.Signer;
+using Nethereum.Util;
+using Web3Unity.Scripts.Library.IPFS;
+using Web3Unity.Scripts.Prefabs;
+
+namespace Scripts.EVM.Token
+{
+    public class Evm
```

Comment:
Class itself can then also be static since all of it's methods are static.

Reply:
good point, updating.

Reply:
done